.github/ISSUE_TEMPLATE/A.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Having trouble getting started?
+about: Please contact us at support@bugsnag.com for assistance with integrating Bugsnag
+  into your application.
+title: ''
+labels: ''
+assignees: ''
+
+---
+Please checkout our [documentation](https://docs.bugsnag.com/performance/integration-guides/go/) for guides, references and tutorials.
+
+If you have questions about your integration please contact us at [support@bugsnag.com](mailto:support@bugsnag.com).
+
+Alternatively, view additional options at [support.md](../SUPPORT.md).
```

Comment:
Hmmm this seems broken too. I think the [support-issue-template-builder](https://github.com/bugsnag/support-issue-template-builder) needs updating so that it's creating a `SUPPORT.md` file, rather than `support.md`.